compiler/src/iree/compiler/Codegen/LLVMCPU/test/pipeline_tests.mlir

```diff
@@ -63,6 +64,16 @@ hal.executable private @check_no_cse {
 //      CHECK:    %[[RES:.+]] = vector.extract %[[DIVF]]
 //      CHECK:    memref.store %[[RES]]
 
+//      REORDERCHECK: func.func @check_no_cse()
+//  REORDERCHECK-NOT:    memref.alloc
+//      REORDERCHECK:    %[[FOR:.+]] = scf.for
+//      REORDERCHECK:    %[[EXTRACT:.+]] = vector.extract %[[FOR]]
+//      REORDERCHECK:    %[[REDUCTION:.+]] = vector.reduction <add>, %[[EXTRACT]]
+//      REORDERCHECK:    %[[INS:.+]] = vector.insertelement %[[REDUCTION]]
+//      REORDERCHECK:    %[[DIVF:.+]] = arith.divf %[[INS]]
+//      REORDERCHECK:    %[[RES:.+]] = vector.extract %[[DIVF]]
+//      REORDERCHECK:    memref.store %[[RES]]
+
```

Comment:
Hmm... why don't we move this test to the split_reduction test below and leave this file intact? This file seems to be testing something unrelated to reductions and we only have `REORDERCHECK` in one of the tests.